### DISCUSSION TITLE
Receiving [RTPS_MSG_OUT Warning] Operation not permitted -> Function send, and unable to select topic

### DISCUSSION THREAD

**Discussion by weithegreat** [eProsima/Fast-DDS#3729] [Q&A]
When pressing "start" on fastdds plugin streaming, the console that launches plotjuggler is printing out warning log
Receiving [RTPS_MSG_OUT Warning] Operation not permitted -> Function send, and unable to select topic
![image](https://github.com/eProsima/fastdds-visualizer-plugin/assets/5653120/4f1051eb-b9f0-453e-91f3-ef40eed47d55)

It's not impacting functionality so far, but I'm curious what I did wrong here.

Regarding my system:
I was running the experiment on a docker container with Ubuntu 20.04.6 LTS.
The eProsima version plotjuggler I used was v1.0.0.
For the built-from-source version I was using: 
plotjuggler fc1cb76ea6a7efacdf159aba0bbef55ec4b46e63, latest as of 05/30/2023 PST
fastdds-visualizer-plugin:265a22e3d9b390d8be149538057619f045b1827e, latest as of 05/30/2023 PST

Regarding the message, I was using static type defined as:
`struct HelloWorld
{
    unsigned long index;
    string message;
};
`
The typeobject generation is enabled and published.

**Comment by jparisu** [comment]
Hi @weigewansui 

It seems like this is a transport warning raised because some message could not be correctly pushed in the socket, or something similar.
This is not related with fastdds-visualizer-plugin, but with Fast DDS. I transfer this issue to them.

**Comment by JLBuenoLopez** [comment]
Hi @weigewansui 

This warning is outputted when something has gone wrong calling to the Asio dependency when sending using UDP transport. If the functionality is not impacted, then you should not be worried. Maybe this message can be demoted to a simple INFO message instead of a Warning. Nevertheless, Fast DDS by default only enables the Error level log message as it is usually the level of interest to a user as this errors prevents the application to work as expected.

As it is not a proper bug, I am moving this ticket to the Q&A discussion forum as per [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md).
